locale/fr/foreman_bootdisk.po

```diff
@@ -256,7 +256,7 @@ msgid "Unable to generate disk %{kind} template: %{error}"
 msgstr "Impossible de générer le modèle %{kind} de disque : %{error}"
 
 msgid "Unable to generate disk template, %{kind} template not found."
-msgstr "Impossible de générer le modèle de disque, le modèle %%{kind} est introuvable."
+msgstr "Impossible de générer le modèle de disque, le modèle%{kind} est introuvable."
```

Comment:
This looks like a regression. I've just applies this change to Transifex so if you pull again it should be good again
```suggestion
msgstr "Impossible de générer le modèle de disque, le modèle %{kind} est introuvable."
```